app/src/main/java/com/example/narutoquiz/data/model/Family.kt

```diff
@@ -0,0 +1,74 @@
+package com.example.narutoquiz.data.model
+
+import com.fasterxml.jackson.annotation.JsonProperty
+
+data class Family(
+    @JsonProperty("adoptive brother")
```

Comment:
JsonProperty tüm alanlar için kullanmak zorunda mı kaldık? Sorun yaşanılan alanlar dışında kaldırsak hata veriyor mu?

Reply:
```
 fun provideRetrofitService(mapper: ObjectMapper): NarutoService {
        return Retrofit.Builder().addConverterFactory(JacksonConverterFactory.create(mapper))
            .baseUrl(BuildConfig.BASE_URL)
            .build()
            .create(NarutoService::class.java)
    }
```
bunun içerisinde jackson tanımladığım için gson kullandığım zaman hata veriyordu